server/src/main/antlr4/com/broadcom/lsp/cobol/core/parser/CobolLexer.g4

```diff
@@ -24,6 +24,7 @@ LEVEL_NUMBER_77 : '77';
 LEVEL_NUMBER_88 : '88';
 
 // keywords
+ABEND : A B E N D;
```

Comment:
All the new tokens should be added to the `cobolWord` rule to allow them to be also variable names.